<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manille</class>
 <widget class="QMainWindow" name="manille">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyManille!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>704</width>
      <height>138</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="card0">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="card1">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="card2">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="card3">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="card4">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="card5">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="card6">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="card7">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>GreyWyvern/cardset/01c.gif</normaloff>GreyWyvern/cardset/01c.gif</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>120</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>290</width>
      <height>440</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="T0">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>140</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>GreyWyvern/cardset/bottom02.gif</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="T1">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>140</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>GreyWyvern/cardset/bottom02.gif</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="T2">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>140</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>GreyWyvern/cardset/bottom02.gif</pixmap>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="T3">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>140</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>GreyWyvern/cardset/bottom02.gif</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="next">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>92</width>
      <height>167</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="trump_label">
       <property name="text">
        <string>Trump:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="trump_disp">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>140</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>GreyWyvern/cardset/bottom02.gif</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
